review-tools/pick-to-branch

```diff
@@ -112,7 +112,7 @@ git checkout $target
 ORIG_TARGET_HEAD=`git show -s --format="%H"`
 git pull --ff-only `git rev-parse --abbrev-ref  @{u} | sed "s|/| |"`
 CHERRYPICKING=1
-git cherry-pick -e -x $id~$num..$id || (git cherry-pick --abort; exit 1)
+git cherry-pick -e -x $id~$num..$id || (echo 'Fix or Ctrl-d to abort' ; read || exit 1)
```

Comment:
This isn't restoring to the pre-cherry pick state, should it?

Reply:
The cleanup trap function will do it.

Reply:
Yeah, the above call of `git cherry-pick --abort` was (meanwhile) redundant.